doc/man3/idset_create.rst

```diff
@@ -106,6 +106,10 @@ IDSET_FLAG_INITFULL
    The idset is created full instead of empty.  If specified with
    IDSET_FLAG_AUTOGROW, new portions that are added are also filled.
 
+IDSET_FLAG_ALLOC_RR
+   Change :func:`idset_alloc` to begin searching for an id after the
+   most recently allocated one, rather than taking the first available.
+
```

Comment:
Should the note about time complexity be added here?

Reply:
Ah, I must have missed the mention when reading the diff, sorry! you can ignore this comment then.